src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -416,6 +420,20 @@ private void updateSupplementaryEvidence(
         CaseDetails exceptionRecordDetails,
         boolean ignoreWarnings
     ) {
+        Optional<String> attachedToCase = getCaseExceptionRecordIsAttachedTo(
+            callBackEvent.exceptionRecordJurisdiction,
+            callBackEvent.exceptionRecordId
+        );
+
+        if (attachedToCase.isPresent()) {
+            log.warn("There has been an attempt to attach an already attached exception record to another case. "
+                + "Exception record ID: {}, attempt to attach to case: {}, already attached to case: {}",
+                callBackEvent.exceptionRecordId,
+                targetCaseCcdRef,
+                attachedToCase.get());
+            return Optional.empty();
+        }
```

Comment:
I haven't spotted it at first, but we should only log warning and progress when the exception record is already attached to the case that we are trying to attach it to. If we're trying to attach it to a different case, I think we should throw an exception, just like we do for supplementary evidence without OCR.